contracts/zap/decollateralize/SolidZapDecollateralize.sol

```diff
@@ -3,12 +3,108 @@ pragma solidity 0.8.18;
 
 import "./BaseSolidZapDecollateralize.sol";
 
+interface SWManager {
+    function bulkDecollateralizeTokens(
+        uint[] calldata batchIds,
+        uint[] calldata amountsIn,
+        uint[] calldata amountsOutMin
+    ) external;
+
+    function getBatchCategory(uint batchId) external view returns (uint);
+}
```

Comment:
Could you explain why you just didn't import existing interface?

Reply:
the issue is with pragma version, where the zap is 0.8.18, but the existing interfaces are 0.8.16, and solc doesn't compile

and instead of duplicating existing interfaces to new files and changing pragma to 0.8.18, it was simpler to just declare an interface here only with the functions getting called on the manager

Reply:
Thanks for explanation